src/Cluster/RedisStrategy.php

```diff
@@ -55,4 +55,15 @@ public function getDistributor()
             'This cluster strategy does not provide an external distributor'
         );
     }
+
+    /**
+     * @inheritDoc
+     */
+    protected function getDefaultCommands()
+    {
+        $commands = parent::getDefaultCommands();
+        $commands['WAIT'] = array($this, 'getKeyFromFirstArgument');
```

Comment:
@nrk Yeah, I was thinking the same. The problem is that there is no way to bypass the hash generator for the cluster strategy. But maybe it makes sense to return a random slot, since Redis is tracking the writes internally for the current client connection (see http://redis.io/commands/WAIT#implementation-details). Does this makes sense to you?